### PR TITLE
OEL-1175: Image and styles added to solve required fields on forms.

### PR DESCRIPTION
**Pull request by Maxfire**
Jira issue(s):
- OEL-1175: Asterisks for required field for forms.

### CODE REVIEW

**Review by drishu — DISMISSED** [review]

File: assets/images/required.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16"><path fill="#EE0000"  d="M0,7.562l1.114-3.438c2.565,0.906,4.43,1.688,5.59,2.35C6.398,3.553,6.237,1.544,6.22,0.447h3.511 c-0.05,1.597-0.234,3.6-0.558,6.003c1.664-0.838,3.566-1.613,5.714-2.325L16,7.562c-2.05,0.678-4.06,1.131-6.028,1.356 c0.984,0.856,2.372,2.381,4.166,4.575l-2.906,2.059c-0.935-1.274-2.041-3.009-3.316-5.206c-1.194,2.275-2.244,4.013-3.147,5.206 l-2.856-2.059c1.872-2.307,3.211-3.832,4.017-4.575C3.849,8.516,1.872,8.062,0,7.562"/></svg>
```
Comment (escuriola) [inline assets/images/required.svg]: No internal files are stored in assets, is a folder for external files or processed/compiled. After talk with @drishu, this file should be stored at resources/images folder.

**Review by escuriola — CHANGES_REQUESTED** [review]

File: resources/sass/components/_form.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
+// form
+// https://getbootstrap.com/docs/5.0/forms/overview/
```
Comment (escuriola) [inline resources/sass/components/_form.scss]: No more links to bootstrap. Remove this line

**Review by escuriola — CHANGES_REQUESTED** [review]

File: resources/sass/components/_form.scss [hunk 3]
```diff
@@ -0,0 +1,18 @@
+// form
```
Comment (drishu) [inline resources/sass/components/_form.scss:1]: so the filename is _form, then it starts with // form and the it says // Custom css for forms. I think that it too much

File: resources/sass/components/_form.scss [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+// Custom css for forms.
+/**
+ * Required field
+ */
+.form-required:after {
+  content: '';
+  vertical-align: super;
+  display: inline-block;
+  background-image: url(images/required.svg);
```
Comment (drishu) [inline resources/sass/components/_form.scss]: this doesn't seem to work
image

**Review by drishu — CHANGES_REQUESTED** [review]

**Review by escuriola — DISMISSED** [review]

File: resources/sass/components/_form.scss [hunk 5]
```diff
@@ -0,0 +1,16 @@
+// form
+
+/**
+ * Required field
+ */
+.form-required:after {
+  content: '';
+  vertical-align: super;
+  display: inline-block;
+  background-image: url('../../images/required.svg');
```
Comment (drishu) [inline resources/sass/components/_form.scss]: are we guessing ? can you please verify before submitting.

Comment (Maxfire) [reply resources/sass/components/_form.scss]: We did, no guessing we checked, anyway fixed again, thanks for your input.

**Review by escuriola — APPROVED** [review]

**Review by drishu — APPROVED** [review]

**Pull request merged**